pkg/image/importer/transport.go

```diff
@@ -0,0 +1,112 @@
+package importer
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
+
+	restclient "k8s.io/client-go/rest"
+)
+
+const (
+	// dockerCertsDir is the directory where Docker stores certificates.
```

Comment:
based on the code it seems like it nests them in directories named by registry host?

Reply:
Yes, and it requires special filename extensions: .key/.cert for certificates, .crt for CA. Docker has documentation for [the directory layout](https://docs.docker.com/engine/security/certificates/#understanding-the-configuration).